config/secrets.development.example.yml

```diff
@@ -1,8 +1,10 @@
 host: 'localhost:3000'
 redis_url: 'redis://127.0.0.1:6379/0'
 
+# TODO: document creating a dev app
```

Comment:
https://github.com/Shopify/shipit-engine/blob/master/docs/setup.md#creating-the-github-app should suffice?

Reply:
Possibly! I carried it over from the old config. I'll remove it.

Reply:
Done :)